extensions/typescript-language-features/package.nls.json

```diff
@@ -62,7 +62,7 @@
 	"configuration.tsserver.experimental.enableProjectDiagnostics": "(Experimental) Enables project wide error reporting.",
 	"typescript.locale": "Sets the locale used to report JavaScript and TypeScript errors. Requires using TypeScript 2.6.0 or newer in the workspace. Default of `null` uses VS Code's locale.",
 	"javascript.implicitProjectConfig.experimentalDecorators": "Enable/disable `experimentalDecorators` for JavaScript files that are not part of a project. Existing jsconfig.json or tsconfig.json files override this setting. Requires using TypeScript 2.3.1 or newer in the workspace.",
-	"configuration.suggest.autoImports": "Enable/disable auto import suggestions. Requires using TypeScript 2.6.1 or newer in the workspace.",
+	"configuration.suggest.autoImports": "Enable/disable auto import suggestions. Requires using TypeScript 3.0 or newer in the workspace.",
```

Comment:
As discussed in #104178, the suggested fix it to remove the entire `Requires using TypeScript 2.6.1 or newer in the workspace.` line

We'd also like the same for all settings that require a version of TypeScript that is less than 3.0